docs/metrics/metrics-operators/quantize.md

```diff
@@ -56,3 +57,11 @@ The `quantize` clause in this metric query sets the time bucket size to 10 minut
 ```sql
 metric=CPU_User cluster=kafka | quantize to 10m using sum drop last
 ```
+
+### Omit time bucket size
+
+In this example we omit the `to INTERVAL` part of the query and let Sumo determine appropriate quantization interval according to [How sumo chooses rollup table and quantization interval](../introduction/metric-quantization.md#how-sumo-chooses-rollup-table-and-quantization-interval). `sum` rollup type will be used to aggregate the metrics in each time bucket.
+
+```sql
+metric=CPU_User cluster=kafka | quantize using sum
```

Comment:
I this case the `sum` of `CPU_User` has a little sense, how about using `avg` in the example?

Reply:
`avg` is the default but `max` could work better.

Reply:
`avg` does not make much sense too as it is the default. I will use non-default `max` which should be more realistic, also for other examples in this doc.